tests/unit/Utopia/Response/Filters/V15Test.php

```diff
@@ -562,39 +562,39 @@ public function testDocumentList(array $content, array $expected): void
         $this->assertEquals($expected, $result);
     }
 
-    /**
-     * @dataProvider createdAtUpdatedAtProvider
-     */
-    public function testDomain(array $content, array $expected): void
-    {
-        $model = Response::MODEL_DOMAIN;
-
-        $result = $this->filter->parse($content, $model);
-
-        $this->assertEquals($expected, $result);
-    }
-
-    /**
-     * @dataProvider createdAtUpdatedAtProvider
-     */
-    public function testDomainList(array $content, array $expected): void
-    {
-        $model = Response::MODEL_DOMAIN_LIST;
-
-        $content = [
-            'domains' => [$content],
-            'total' => 1,
-        ];
-
-        $expected = [
-            'domains' => [$expected],
-            'total' => 1,
-        ];
-
-        $result = $this->filter->parse($content, $model);
-
-        $this->assertEquals($expected, $result);
-    }
+    // /**
+    //  * @dataProvider createdAtUpdatedAtProvider
+    //  */
+    // public function testDomain(array $content, array $expected): void
+    // {
+    //     $model = Response::MODEL_DOMAIN;
+
+    //     $result = $this->filter->parse($content, $model);
+
+    //     $this->assertEquals($expected, $result);
+    // }
```

Comment:
why did we comment these out?

Reply:
It needs `Response::MODEL_DOMAIN` which no longer exists. I commented it out in like 2 or 3 migration files

Reply:
Let's just delete instead of commenting them out